src/ReactiveUI/Cocoa/CommonReactiveSource.cs

```diff
@@ -330,7 +330,19 @@ private void SubscribeToSectionInfoChanges(int sectionInfoId, IReadOnlyList<TSec
                                             });
                                     }
 
-                                    this.pendingChanges.Add(Tuple.Create(y.Section, new PendingChange(y.Change)));
+
+                                    if (y.Change.OldItems?.Count > 1 && y.Change.Action == NotifyCollectionChangedAction.Remove) 
+                                    { 
+                                        foreach(var item in y.Change.OldItems) {
+
+                                            var args = new NotifyCollectionChangedEventArgs(NotifyCollectionChangedAction.Remove, item, y.Change.OldStartingIndex);
+                                            this.pendingChanges.Add(Tuple.Create(y.Section, new PendingChange(args)));
+                                        }
+                                    } 
+                                    else {
```

Comment:
Not applicable anymore but  I didn't notice I did that... I'm doing more right now on VS for Mac since the latest VS 17  / 15.2 is pretty messed up and VS for Mac really likes fighting me it seems when I'm trying to format my code.